composer.json

```diff
@@ -1,11 +1,13 @@
 {
   "license": "GPL-3.0+",
   "require": {
+    "ext-intl": "*",
```

Comment:
Instead of requiring intl (which might require someone to spend half an hour upgrading their PHP), let's just check for the existence of NumberFormatter. After all, number formatting isn't critical to the tool's functionality:
```
if ( class_exists( 'NumberFormatter' ) ) {
	$formatter = new \NumberFormatter( $locale, \NumberFormatter::DECIMAL );
	// Get separator symbols (include decimal as it is required and we might use it at some point)
	$decimal = $formatter->getSymbol( \NumberFormatter::DECIMAL_SEPARATOR_SYMBOL );
	$thousands = $formatter->getSymbol( \NumberFormatter::GROUPING_SEPARATOR_SYMBOL );
} else {
	$decimal = '.';
	$thousands = ',';
}